### ISSUE TITLE
Use nbcollection?

### ISSUE THREAD

**Issue by pllim** [spacetelescope/notebooks#134]
https://github.com/adrn/nbcollection/

**Comment by jbcurtin** [comment]
yeah, that is my understanding. I'm writing most of the logic to handle multi-repo CI-Integration in `adrn/nbcollection`

I came back to this repo to find the TOC issue and post it into,
https://github.com/adrn/nbcollection/issues/5
